app-office/libreoffice/additional-files/libreoffice.rdef.in

```diff
@@ -30,13 +30,22 @@ resource file_types message {
 };
 
 resource vector_icon {
-	$"6E63696601050003020AC115B38FC115B38FC0C6B340BFB920C02920BFB920B6"
-	$"2E20B62E20B5BE20B562B3CBB562B35BB562B3CBB562CBA7B562CBA7B562CC17"
-	$"B62ECC73B5BECC73B62ECC73C944CC73C944CC73C9B4CC73CA10CBA7CA10CC17"
-	$"CA10CBA7CA10BD56CA10BD56CA10BCE6C980BBFBC9CFBC4AC980BBFB0608D27F"
-	$"C879CADCB6FAB497BFB4B497BFB4B497BFC6B49ABFF9B4B3BFE9B4A8BFF9B4B3"
-	$"C85CBD17C85CBD17C867BD26C8793AC876BD49C8793ACADC0206C94420C9B420"
-	$"C94420C47F20C47F20C40F20C443B38FC3F4B340C443B38FC980B8CCC980B8CC"
-	$"C9CFB91CCA10B891CA10B901CA10B891CA10B3CBCA10B3CBCA10B35B010A0003"
-	$"00010200"
-};
+	$"6E636966160300A50005E105FF020004022BE7FFE7F4008C0003008C00020004"
+	$"020EE7FFE7F3008C000301000003010000030100000301000003010000030100"
+	$"0003010000030100000301000003010000030100000301000003010000030100"
+	$"0001010000C501010000C5160A0A25273A234F274F2D3E343E553A5636553634"
+	$"252D0A043E34522B52533E5C0A04232C3634365C23530A0425273A234F273A2F"
+	$"0A063A563E553E344F2C4F273A2F0A063A5636553634252D25273A2F0A044F3D"
+	$"414341554F4F080248404852080241464E40080241474E410A0E365C2353232C"
+	$"252D25273A234F274F2C522B52533E5C3E553A5636550803232C373537550803"
+	$"4F2C3D353D55080325273A2F4F2708023A303A5608023E3D44320802363D3032"
+	$"08023342253B08022540334708022545334C0806365D36553A573E553E5D3C5C"
+	$"0A093E5D535353416045544B5B4D57506053495D170A000100000A010101000A"
+	$"020102000A030103000A040104000A050105000A06010630201E01178100040A"
+	$"07010730201E01178100040A08010830202101178100040A0901093020260117"
+	$"8100040A0A010A100217810004158100040A0B010B1001178100040A0C010C10"
+	$"01178100040A0D010D1001178100040A0E010E1001178100040A0F010F100117"
+	$"8100040A1001101001178100040A1101111001178100040A1201121001178100"
+	$"040A1301131001178100040A13011330202501178100040A140114000A150115"
+	$"00"	
+	};
```

Comment:
Cool! Thanks @roired for your PR! (I'll have a look at the new icon asap.)

Hmm, I think the preferred style for the closing `}` is without a tab:
```diff
- 	};
+};
```
So, shall we drop the tab in line 51?

Reply:
Good day,
I think the tab is a mistype from the copy-paste due to this being my first attempt at a PR.
Regards,
RR

Reply:
so sure, can drop that tab :)